src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -149,6 +156,16 @@ We support
             }
             else
             {
+                string rawRequirementType = configurationSection[RequirementTypeKeyword];
+
+                if (!string.IsNullOrEmpty(rawRequirementType))
+                {
+                    if (!Enum.TryParse(rawRequirementType, out requirementType))
```

Comment:
This needs to be case insensitive.

Reply:
What would be your reasoning that it would need to be case insensitive?

Reply:
We have case insensitive comparison while parsing other feature flag fields, like feature name, filter name, etc. Just want to maintain the same behavior for requirement type.
I imagine people might try to use RequirementType "ALL", "All" or "all" interchangeably.

Reply:
Yes, I think it would stick out if this was case sensitive given the other examples you mentioned.